### PR TITLE
Update getting_started.md

### PR DESCRIPTION
**Pull request by lboix**
**Adding instructions for enabling CWS module when Datadog Agent has been installed manually as a Daemonset**

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/cloud_workload_security/getting_started.md
Comment (lebauce) [inline content/en/security_platform/cloud_workload_security/getting_started.md:85]: We also need:
```
- name: HOST_ETC
  value: /host/root/etc
```

File: content/en/security_platform/cloud_workload_security/getting_started.md
Comment (lebauce) [inline content/en/security_platform/cloud_workload_security/getting_started.md:98]: Is the indentation correct ?

Comment (lboix) [reply content/en/security_platform/cloud_workload_security/getting_started.md:100]: Hello @lebauce fron my point of view yes : I copy-paste the manifest I used in production and strip the non related. Of course someone who wants to use this snippet will have to copy-paste things and respect .yaml indentation. I can add a warning message if you want, let me know

**Review by n06 — APPROVED** [review]
Hi @lboix thank you for making this pull request! 

I will defer to @lebauce for technical accuracy, but once that's satisfied this LGTM!

File: content/en/security_platform/cloud_workload_security/getting_started.md
Comment (n06) [inline content/en/security_platform/cloud_workload_security/getting_started.md:86]: @lebauce should this be a volume mount or is it in the right section?

File: content/en/security_platform/cloud_workload_security/getting_started.md
Comment (lboix) [reply content/en/security_platform/cloud_workload_security/getting_started.md:100]: Let us know if it's OK for you here @lebauce 👍

**Comment by lebauce** [comment]
@lboix Apologies for taking so long to answer.

We initially asked the host root to be mounted to /host as it was convenient but it's not super nice. I would prefer to mount only the actually required paths.

```
/sys => /host/sys
/proc => /host/proc
/usr/lib/os-release => /host/usr/lib/os-release
/etc/passwd => /etc/passwd
/etc/group => /etc/group
```

Regarding the environment variable, you don't need to specify HOST_ROOT neither HOST_ETC.

**Comment by kayayarai** [comment]
We had enough docs changes since this was created that I can't merge it. I'll close this one and am reopening it as https://github.com/DataDog/documentation/pull/13040

**Comment by kayayarai** [comment]
Recreating, because it got too stale

**Pull request closed without merging**
